packages/geolocation/README.md

```diff
@@ -23,7 +23,7 @@ yarn add @solid-primitives/geolocation
 
 ### createGeolocation
 
-Used to fetch current geolocation data as a resource.
+Used to fetch current geolocation data as a resource. This primitive uses `createResource` to return a location, so `loading`, `error`
```

Comment:
This sentence seems incomplete, and the code snippet below doesn't have `loading` or `error` used.